x-pack/plugins/fleet/public/components/agent_enrollment_flyout/confirm_agent_enrollment.tsx

```diff
@@ -139,10 +141,17 @@ export const ConfirmAgentEnrollment: React.FunctionComponent<Props> = ({
           color="primary"
           iconType={EuiLoadingSpinner}
           title={
-            <FormattedMessage
-              id="xpack.fleet.agentEnrollment.loading.listening"
-              defaultMessage="Listening for agent..."
-            />
+            isLongEnrollment ? (
+              <FormattedMessage
+                id="xpack.fleet.agentEnrollment.loading.listeningcsp"
```

Comment:
nit: as we change to `longEnrollment` makes sense to update the translation key to match that as well